get/install.bash

```diff
@@ -100,24 +137,53 @@ get_n_extract() {
         
 		;;
 	Linux)
-        STATIC_RELEASE_URL="https://github.com/tuzig/webexec/releases/download/v$LATEST_VERSION/webexec_${LATEST_VERSION}_$(uname -s | tr '[:upper:]' '[:lower:]')_$ARCH.tar.gz"
-       curl -sL "$STATIC_RELEASE_URL" | tar zx --strip-components=1
-       ./webexec init
+        BALL_NAME="webexec_${LATEST_VERSION}_$(uname -s | tr [:upper:] [:lower:])_$ARCH.tar.gz"
+        STATIC_RELEASE_URL="https://github.com/tuzig/webexec/releases/download/v$LATEST_VERSION/$BALL_NAME"
+        curl -sL "$STATIC_RELEASE_URL" -o $1/$BALL_NAME
+        tar zx --strip-components=1 -C $1 < $1/$BALL_NAME 
 	esac
 }
+
 do_install() {
 	init_vars
 	checks
 
-	tmp=$(mktemp -d)
+	sh_c='sh -c'
+	if [ "$user" != 'root' ]; then
+		if command_exists sudo; then
```

Comment:
Given you have 3 possible outcomes:
1. system has sudo, but user has no permissions
2. system has sudo, user has permissions
3. system has no sudo

I think the best strategy would be:
check a dummy `sudo` command's return value, e.g
`sudo true ; echo $?`
a value of 0 indicates both having a `sudo` command, and a user with sudo permissions.
a value of 1 indicates either no `sudo` command or no user permissions.
You might want `/bin/true` instead of `true` to avoid counting on `$PATH`, but perhaps it's a non-issue.
Only if the above fails you want to try `su`.

Ofcourse if you want to be super-nice , you can separate the cases, and report different error messages for no sudo, and for lack of sudo permissions.

Reply:
that's a great idea. I copied this code (and as much as possible ;-) from https://github.com/docker/docker-install/blob/master/install.sh and I think you should open a PR there and shave that yack.

Reply:
that's funny. I coded it and found out it's not such a great idea. If I'm a sudoer, all this change does is add another passwd I have to key in. 
On those rare cases when the user is not on the sudoer list, they'll get:

```
jrandomhacker is not in the sudoers file.  This incident will be reported.
```